src/lib.zig

```diff
@@ -4343,7 +4371,27 @@ pub const Lua = opaque {
     /// Works with ints, floats, booleans, structs,
     /// tagged unions, optionals, and strings
     pub fn pushAny(lua: *Lua, value: anytype) !void {
-        switch (@typeInfo(@TypeOf(value))) {
+        const T = @TypeOf(value);
+        const type_info = @typeInfo(T);
+
+        if (type_info == .@"struct" or type_info == .@"union" or type_info == .@"enum") {
+            if (@hasDecl(T, "ziglua_pushAny")) {
```

Comment:
I'm not entirely sold on the `ziglua_pushAny` and `ziglua_toAny` naming convention because it uses mixed camel and snake case.

std.json uses `jsonParse` and `jsonStringify` to accomplish this purpose.

The simple solution would be to use`zigluaPushAny` and `zigluaToAny`

Alternatively, `toLua` and `fromLua` might make more sense? ie, you are converting a value to lua or to zig from lua? I'm not sure on this.

@natecraddock any thoughts?

Reply:
I like this, `toLua` and `fromLua`.

Reply:
I also like toLua and fromLua